lib/semantics/check-omp-structure.cc

```diff
@@ -319,6 +373,17 @@ void OmpStructureChecker::Enter(const parser::OmpClause::Ordered &x) {
             "a constant positive integer expression"_err_en_US);
       }
     }
+
+    // 2.8.3 Loop SIMD Construct Restriction
+    if (GetContext().directive == OmpDirective::DO_SIMD ||
+        GetContext().directive == OmpDirective::PARALLEL_DO_SIMD) {
+      auto dir{EnumToString(GetContext().directive)};
+      std::replace(dir.begin(), dir.end(), '_', ' ');
```

Comment:
You should extract this into a function, in part so that the reader can tell why you're doing this string manipulation (either from the name of the function or a comment on it).

We use `AsFortran` for functions that return a representation of something as it can appear in Fortran source, so that seems appropriate here.  Maybe `std::string AsFortran(OmpDirective)` or `std::string ContextDirectiveAsFortran()`.

Reply:
Addressed in update commit. Thanks.